script.js

```diff
@@ -1,67 +1,75 @@
-let book = [];
-
 const givenName = document.querySelector('#name');
 const givenAuthor = document.querySelector('#author');
 const btnClass = document.querySelector('#Button');
 const listOfName = document.querySelector('#listOfName');
 
-function updateStorage() {
-  localStorage.setItem('books', JSON.stringify(book));
-}
-function removeBook(id, c = true) {
-  if (c === false) {
-    return;
+class BookList {
+  constructor(book) {
+    this.book = book;
```

Comment:
[OPTIONAL] Because the `book` variable is an array of books, I recommend using the name `books` instead of the name `book` to make your code more readable.

Reply:
Nice catch, I agree. 👍🏽